Update connections.md (#16747)
yugabyte/yugabyte-db: docs/content/preview/launch-and-manage/monitor-and-alert/metrics/connections.md

Connection metrics represent the cumulative number of connections to the YSQL backend per node. This includes various background connections, such as checkpointer, active connections count that only includes the client backend connections, newly established connections, and connections rejected over the maximum connection limit.

Connection metrics are only available in Prometheus format.

The following table describes key connection metrics.

| Metric | Unit | Type | Description |
| :------ | :--- | :--- | :---------- |
| `yb_ysqlserver_active_connection_total` | connections | counter | The number of active client backend connections to YSQL server. If a client connection is executing a statement, it is considered an active connection. Any client connection not executing a statement is considered an idle connection.|
| `yb_ysqlserver_connection_total` | connections | counter | The total number of all connections to YSQL, which includes active connections, idle connections, and background connections. |

> | `yb_ysqlserver_max_connection_total` | connections | counter | The total number of  connections that a YSQL server can support at given time. The default is 100 and value can be changed using `max connection` GUC flag |

Reviewer: ```suggestion
| `yb_ysqlserver_max_connection_total` | connections | counter | The maximum number of concurrent connections that a YSQL server can support at any given time. The default is 100. This value can be changed using the `--ysql_max_connections` YB-TServer flag. |
```

> | `yb_ysqlserver_max_connection_total` | connections | counter | The total number of  connections that a YSQL server can support at given time. The default is 100 and value can be changed using `max connection` GUC flag |

Reviewer: I'm guessing - not sure what GUC flag refers to otherwise
ref https://docs.yugabyte.com/preview/reference/configuration/yb-tserver/#ysql-max-connections

> | `yb_ysqlserver_max_connection_total` | connections | counter | The maximum number of concurrent connections that a YSQL server can support at any given time. The default is 100. This value can be changed using the `--ysql_max_connections` YB-TServer flag. |

Reviewer: ```suggestion
| `yb_ysqlserver_max_connection_total` | connections | counter | The maximum number of concurrent connections that a YSQL server can support at any given time. The default is 300. This value can be changed using the `--ysql_max_connections` YB-TServer flag. |
```

> | `yb_ysqlserver_connection_over_limit_total` | connections | counter | The number of connection requests rejected by YSQL server over the maximum connection limit set based on yb_ysqlserver_max_connection_total.  |

Reviewer: ```suggestion
| `yb_ysqlserver_connection_over_limit_total` | connections | counter | The number of connection requests rejected by the YSQL server over the maximum connection limit, based on `yb_ysqlserver_max_connection_total`.  |
```
